odoo/addons/test_new_api_extension/models/test_fields.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+class DummyModel(models.Model):
+    _name = 'test_new_api.dummy'
```

Comment:
This is no model extension!
```suggestion
    _inherit = 'test_new_api.dummy'
```

Reply:
Indeed :facepalm: